config/cors.php

```diff
@@ -10,37 +10,20 @@
  */
 
 return [
-
     /*
      |--------------------------------------------------------------------------
-     | Laravel CORS Defaults
+     | Laravel CORS
      |--------------------------------------------------------------------------
      |
-     | The defaults are the default values applied to all the paths that match,
-     | unless overridden in a specific URL configuration.
-     | If you want them to apply to everything, you must define a path with *.
-     |
-     | allowedOrigins, allowedHeaders and allowedMethods can be set to ['*']
-     | to accept any value, the allowed methods however have to be explicitly listed.
+     | allowedOrigins, allowedHeaders and allowedMethods can be set to array('*')
+     | to accept any value.
      |
      */
-    'defaults' => [
-        'supportsCredentials' => true,
-        'allowedOrigins'      => ['*'],
-        'allowedHeaders'      => [],
-        'allowedMethods'      => [],
-        'exposedHeaders'      => [],
-        'maxAge'              => 0,
-        'hosts'               => [],
-    ],
-
-    'paths' => [
-        'api/v1/*' => [
```

Comment:
I added the Cors middleware to the `api` middleware group. I don't know if that fixes it 100%?